common/src/main/java/org/opensearch/ml/common/transport/model/MLUpdateModelInput.java

```diff
@@ -45,25 +47,29 @@ public class MLUpdateModelInput implements ToXContentObject, Writeable {
     private String version;
     private String name;
     private String modelGroupId;
+    private Boolean isEnabled;
+    private MLRateLimiter modelRateLimiterConfig;
     private MLModelConfig modelConfig;
-    private Connector connector;
+    private Connector updatedConnector;
     private String connectorId;
-    private MLCreateConnectorInput connectorUpdateContent;
+    private MLCreateConnectorInput connector;
```

Comment:
This part was for new connector content. From the name this line means the original connector content , as I see line 53 means the new connector content `private Connector updatedConnector;` .   Is the understanding correct ? Why switch the meaning of these two fields?

Reply:
Discussed this topic offline